isthisstockgood/templates/searchbox.html

```diff
@@ -4,11 +4,11 @@
       <div class="col-md-3 nopadding">
         <input
             type="text"
+            autofocus
```

Comment:
Causes the search field to get activated automatically on each visit to the site (including page refresh).

@mrhappyasthma, if you see problems with this behaviour, please let me know! :)